psalm.xml

```diff
@@ -20,6 +20,7 @@
 
     <issueHandlers>
         <RedundantConditionGivenDocblockType errorLevel="suppress"/>
+        <RedundantCastGivenDocblockType errorLevel="suppress"/>
```

Comment:
Is this error triggered at too many places? If it's only for a few then it's better to update the baseline file instead.

Reply:
No, not too many.
I just dont think that it is worth being reported at all, those are usually just nice in 4.3 where we cannot "actually promise" the value, so the cast just makes sure the documented type coming in is the one we expect later on.
In 5.x this can be removed then as we will have types in place to assert it for us. Until then an extra cast might not even be extra, and doesn't hurt in the other case IMO.

Reply:
Agreed, keep for 4.3 and remove for 5.0.